src/mpi/errhan/errnames.txt

```diff
@@ -1409,9 +1409,9 @@ is too big (> MPIU_SHMW_GHND_SZ)
 
 ## MPI-3 Mprobe routines
 **mpi_improbe: MPI_Improbe failed
-**mpi_improbe %d %d %C %p %p %p: MPI_Improbe(source=%d, tag=%d, comm=%C, flag=%p, message=%p, status=%p)
+**mpi_improbe %i %t %C %p %p %p: MPI_Improbe(source=%i, tag=%t, comm=%C, flag=%p, message=%p, status=%p)
```

Comment:
Commit subject says mrecv and imrecv, but changes are for mprobe and improbe.

Reply:
I'll fix it.